sys-libs/ncurses/ncurses6-6.1.recipe

```diff
@@ -0,0 +1,217 @@
+SUMMARY="New curses library"
+DESCRIPTION="The Ncurses (new curses) library is a free software emulation of \
+curses in System V Release 4.0, and more. It uses Terminfo format, supports \
+pads and color and multiple highlights and forms characters and function-key \
+mapping, and has all the other SYSV-curses enhancements over BSD Curses.
+The ncurses code was developed under GNU/Linux. It has been in use for some \
+time with OpenBSD as the system curses library, and on FreeBSD and NetBSD as \
+an external package. It should port easily to any ANSI/POSIX-conforming UNIX. \
+It has even been ported to OS/2 Warp!
+The distribution includes the library and support utilities, including a \
+terminfo compiler tic, a decompiler infocmp, clear, tput, tset, and a \
+termcap conversion tool captoinfo. Full manual pages are provided for the \
+library and tools."
+HOMEPAGE="https://www.gnu.org/software/ncurses/ncurses.html"
+COPYRIGHT="1998-2018 Free Software Foundation, Inc."
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://ftpmirror.gnu.org/ncurses/ncurses-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/ncurses/ncurses-$portVersion.tar.gz"
+CHECKSUM_SHA256="aa057eeeb4a14d470101eff4597d5833dcef5965331be3528c08d99cebaa0d17"
+SOURCE_DIR="ncurses-$portVersion"
+PATCHES="ncurses-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64 arm"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= 6.1"
+portVersionCompat="$portVersion compat >= 5"
```

Comment:
```diff
-portVersionCompat="$portVersion compat >= 5"
+portVersionCompat="$portVersion compat >= 6"
```

Reply:
For `ncurses6-6.0` the `cmd:` entries have `compat >= 5`, so I thought we should keep them, but I might be wrong.

Reply:
The meaning of `compat` is to signify the latest version that the versioned entry is backward compatible to, so I think using `compat >= 6` should be safer, but I don't really know for sure if it's totally backward compatible with version 5.

Reply:
I used `find` with `-exec grep -E -l 'cmd:captoinfo|cmd:clear|cmd:infocmp|cmd:infotocap|cmd:reset|cmd:tabs|cmd:tic|cmd:toe|cmd:tput|cmd:tset' {} \;` on all of our recipes and the only result (other than the recipes for ncurses) is `dev-util/fenris/fenris-0.07.recipe` which has a `cmd:tput` in `BUILD_PREREQUIRES` (but without any condition on the version.)

@extrowerk, @alaviss: This means you were both right: We can use `compat >= 6`.
(Sorry for my previous suggestions to use `compat >= 5` :)

```diff
-portVersionCompat="$portVersion compat >= 5"
+portVersionCompat="$portVersion compat >= 6"
```